website/package.json

```diff
@@ -11,8 +11,8 @@
         "build": "astro build",
         "preview": "astro preview",
         "astro": "astro",
-        "check-format": "eslint --cache \"./**/*.{ts,tsx,astro}\" && prettier --ignore-path \"../.gitignore\" --check \"./**/*.{ts,tsx,json,astro,cjs,mjs,css,scss}\"",
-        "format": "eslint --cache \"./**/*.{ts,tsx,astro}\" --fix && npm run format-fast",
+        "check-format": "eslint --cache . && prettier --ignore-path \"../.gitignore\" --check \"./**/*.{ts,tsx,json,astro,cjs,mjs,css,scss}\"",
+        "format": "eslint --cache . --fix && npm run format-fast",
```

Comment:
I think this caused #3461, it's a shame one can't comment inside the package.json - given one can't comment to explain a certain construct it's sometimes better not to change even if one doesn't understand why something is done a certain way, there might be a reason 🙃